test/unit/test-performance.js

```diff
@@ -687,3 +687,119 @@ describes.realWin('performance with experiment', {amp: true}, env => {
     });
   });
 });
+
+describes.realWin('performance with firstInput events', {amp: true}, env => {
+  describe('should forward firstInput events as a fid tick event', () => {
+    it('before performance service registered', () => {
+      // Pretend that the EventTiming API exists.
+      env.win.PerformanceEventTiming = true;
```

Comment:
This property gates whether the EventTiming API is enabled or not. I though to stub it so it is automatically cleaned up after the test, but it looks like Sinon does not support this for non-existent object properties.

Reply:
try
```
env.sandbox.stub(env.win, 'PerformanceObserver', stub)
```

we create `sinon.sandbox` object for each window context, which will be reset for each test case